shell/components/CommunityLinks.vue

```diff
@@ -131,10 +132,12 @@ export default {
         </a>
       </div>
     </SimpleBox>
-    <modal
+    <app-modal
```

Comment:
A `portal-target` will only render the contents of a single portal at a time. Unless we specify otherwise, only the first modal will display its content.